monai/transforms/croppad/array.py

```diff
@@ -910,16 +910,18 @@ def randomize(
         image: Optional[np.ndarray] = None,
     ) -> None:
         self.spatial_size = fall_back_tuple(self.spatial_size, default=label.shape[1:])
+        fg_indices_: np.ndarray
```

Comment:
May I know why the data type is still `numpy array`? I think you added `Tensor` support?

Thanks.

Reply:
I've updated the helper functions, but not actually updated any of the transforms. Hence, the transforms expect numpy input, and therefore expect numpy output from these helper functions. This PR will then enable me to start updating the transforms that use the helper functions.

Reply:
@Nic-Ma thanks for the suggestion, the example script now runs both normally and with `EnsureTyped` after `LoadImaged`.

Reply:
I see your point, thanks!